File: week1/tarzaniuss-cs50-problems-2024-x-mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;

    do
    {
        height = get_int("Height:");
    }
    while (height > 8 || height < 1);

    for (int i = 0; i < height; i++)
    {
        for (int o = 1; o < height - i; o++)
        {
            printf(" ");
        }
        for (int g = 0; g < i + 1; g++)
        {
            printf("#");
        }
        printf("  ");
        for (int g = 0; g < i + 1; g++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: week6/tarzaniuss-cs50-problems-2025-x-sentimental-mario-more/mario.py
from cs50 import get_int

while (True):
    number = get_int("Height: ")
    if number < 1 or number > 8:
        continue
    else:
        for i in range(1, number+1):
            for a in range(i, number):
                print(" ", end="")

            for b in range(i):
                print("#", end="")

            print("  ", end="")

            for c in range(0, i):
                print("#", end="")

            print()
    break
